package ssp;

//不管是存数据库还是做传输，都存成二进制流的形式
message UserInfoBase {
    //如果服务端升级了客户端没有升级那么可以通过这个强制升级
    optional uint32 ver = 1;
    optional uint32 user_id = 2;
    optional string user_name = 3;
    optional string nick_name = 4;
    optional uint32 reg_time = 5;
    optional uint32 from = 6;
    optional uint32 login_time = 7;
    optional uint32 last_login_time = 8;
    optional uint32 fresh_time = 9;
    optional string password = 10;
    optional uint32 logout_time = 11;
    optional uint32 db_flag = 12;
}
